__tests__/util/mock.ts

```diff
@@ -1,26 +1,24 @@
 import { EventEmitter2 } from 'eventemitter2';
 import { mocked } from 'ts-jest/utils';
 import { IMatcher } from '../../src/matcher';
-import { IObserver } from '../../src/observers';
+import { AbstractObserver } from '../../src/observers';
 
 const emptyFn = () => void 0;
 
 export const getMockedObserver = () => {
-  return mocked<IObserver>({
+  return mocked<any>({
```

Comment:
can we remove any?

Reply:
we cannot remove it due to its have some protected function in it and I cannot mock it directly.